conf/uppmax-modules.config

```diff
@@ -37,6 +37,21 @@ process {
   // if you have your own installation of MultiQC outside of the environment module system.
   // eg: Add the line: process.$multiqc.module = []
   $multiqc.module = ['bioinfo-tools', 'MultiQC/1.3']
+  $get_software_versions.module = [
+    'bioinfo-tools',
+    'FastQC/0.11.5',
+    'HISAT2/2.1.0',
+    'MultiQC/1.3',
+    'picard/2.0.1',
+    'preseq/0.1.0'
```

Comment:
You are missing a comma here @ewels

Reply:
![sadness](http://cowgirlgoods.typepad.com/.a/6a00e54fd9310e88340168e658dda4970c-pi)

Reply:
Ah, yes now I remember one of the resons why we didn't so this from the beginning:

Reply:
Shouldn't need R here I think. I'll remove.